crates/matrix-sdk-base/src/response_processors/verification.rs

```diff
@@ -13,12 +13,59 @@
 // limitations under the License.
 
 use matrix_sdk_crypto::OlmMachine;
-use ruma::{events::AnySyncMessageLikeEvent, RoomId};
+use ruma::{
+    events::{
+        room::message::MessageType, AnySyncMessageLikeEvent, AnySyncTimelineEvent,
+        SyncMessageLikeEvent,
+    },
+    RoomId,
+};
 
 use super::Context;
 use crate::Result;
 
-pub async fn verification(
+/// Process the given event as a verification event if it is a candidate. The
+/// event must be decrypted.
+pub async fn process_if_candidate(
```

Comment:
`process_if_relevant`? `process` ? Not sure these are better. In other places we just have names like `handle`, even though the method filters and only handles if relevant.

Reply:
I like `process_if_relevant`. Have changed to that.